bulbs/api/views.py

```diff
@@ -602,6 +600,18 @@ def group_count(self, request, **kwargs):
         return Response(dict(count=qs.count()))
 
 
+class SendContributorReport(viewsets.GenericViewSet):
+    """Send contribution report email to all relevant ."""
+
+    serializer_class = ContributorReportSerializer
+    permission_classes = [IsAdminUser, CanEditContent]
+
+    def create(self, request, *args, **kwargs):  # NOQA
```

Comment:
We should just disable that lint check altogether to avoid cluttering everywhere with NOQA (every time I see a NOQA while reading the code I have to puzzle out why it's there)